spec/features/basket_2.txt

```diff
@@ -0,0 +1,2 @@
+1 imported box of chocolates at 10.00
+1 imported bottle of perfume at 47.50
```

Comment:
You have repeated this file here and in the root folder. Normally these kinds of files should be all under a descriptive directory name (`input_baskets`?... naming is hard 😅)